api/functions/auth_functions.py

```diff
@@ -0,0 +1,71 @@
+from sqlalchemy.orm import load_only
+from functions.orm_to_dict import orm_to_dict
+from models import Organizations, User_affiliations
+from manage import app
+
+
+def is_super_admin(user_role):
+    """
+    :param user_role: list of dicts containing user roles
+    :return: Returns true or false based on if this user is the given role
+    """
+    user_id = user_role[0]['user_id']
+    with app.app_context():
+        user_is_super_admin = User_affiliations.query \
+            .filter(User_affiliations.user_id == user_id) \
+            .filter(User_affiliations.permission == 'super_admin') \
+            .first()
+    if user_is_super_admin:
```

Comment:
Since this is a DB query, should there be a `is not None:` check?  Or is that implied with what you already have here?

Reply:
It is implied, beautiful feature of python if statements